src/main/java/ch/hevs/cloudio/endpoint/CloudioLog.java

```diff
@@ -0,0 +1,38 @@
+package ch.hevs.cloudio.endpoint;
+
+class CloudioLog{
```

Comment:
Name is not very descriptive, how about CloudioLogMessage?

Reply:
True, will change that.